ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -108,6 +108,10 @@ port_profiles:
         # Port-profile name, to inherit configuration.
         profile: < port_profile_name >
 
+        # Administrative state | optional - default is true
+        # setting to false will set port to 'shutdown' in intended configuration
```

Comment:
Thank you, your modifications look good!
As per the issue, we also need to apply shutdown to the port-channel interface.

Reply:
Please make this a separate knob that would apply to the port-channel interface only.

Reply:
- added support for 'enabled' key under adapter.port_channel
- changed jinja template, molecule unit test and documenation